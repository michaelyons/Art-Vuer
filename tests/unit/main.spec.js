import { shallowMount } from "@vue/test-utils";
import Main from "@/main.js";

describe("Main.js", () => {
  it("matches snapshot", () => {
    const wrapper = shallowMount(Main);
    expect(wrapper).toMatchSnapshot();
  });
});
