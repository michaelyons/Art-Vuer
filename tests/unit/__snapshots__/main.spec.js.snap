// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main.js matches snapshot 1`] = ``;
